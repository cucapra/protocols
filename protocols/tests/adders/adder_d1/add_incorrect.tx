// ARGS: --verilog adders/adder_d1/add_d1.v --protocol adders/adder_d1/add_d1.prot
// RETURN: 101
add_incorrect(1, 2, 3);
add_incorrect(4, 5, 9);
