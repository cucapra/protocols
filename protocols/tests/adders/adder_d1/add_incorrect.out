error: Thread 0 attempted conflicting assignment to 'a': current=ValueOwned(00000000000000000000000000000001), new=ValueOwned(00000000000000000000000000000100)
   ┌─ adders/adder_d1/add_d1.prot:19:3
   │
19 │   DUT.a := a;
   │   ^^^^^^^^^^^ Thread 0 attempted conflicting assignment to 'a': current=ValueOwned(00000000000000000000000000000001), new=ValueOwned(00000000000000000000000000000100)

